package Durga.Exception;

 

public class Testcasethrowablecase3 
{
	public static void main(String [] args)
	{

	try
	{
		System.out.println("Hello");
	}
	catch  ( Exception  e)
	{
	}
}
}


